source/stylesheets/refills/_centered-navigation.scss

```diff
@@ -198,6 +201,7 @@
       }
     }
 
+    &:focus > .submenu,
```

Comment:
We want to target just the first level so that we don't expand sub-sub-menus. Any other way of doing this you think is better?